### PR TITLE
quay-app template - update apiVersions and remove default readOnly

### PR DESCRIPTION
**Pull request by maorfr**
cc @josephschorr @tparikh

### CODE REVIEW

File: deploy/openshift/quay-app.yaml [hunk 1]
```diff
@@ -213,7 +213,6 @@ objects:
           - containerPort: 8443
           volumeMounts:
           - name: configvolume
-            readOnly: false
```
Comment (tparikh) [inline deploy/openshift/quay-app.yaml:216]: Why is this changed needed?

Comment (maorfr) [reply deploy/openshift/quay-app.yaml:216]: openshift removes this field, causing qontract-reconcile to think that it should apply again.
no harm in keeping it, just makes sense to me to not have fields with default values that are removed.

**Comment by tparikh** [comment]
LGTM
